nio.h

```diff
@@ -74,6 +74,7 @@ typedef struct {
 typedef struct {
     u_int type;
     void *dptr;
+    char desc[512];
```

Comment:
I'm not sure that we never have case where 512 is enough (Windows interface with weird long name)